docs/en/observability/create-alerts.asciidoc

```diff
@@ -69,6 +69,66 @@ list an alert on the {observability} Alerts page.
 Only alerts generated by rules relating to Logs, Metrics, Uptime, and APM
 can be viewed on the Alerts page.
 
+[discrete]
+[[create-alerts-configure]]
+== Configure alerts
+
+You may want to disable writing to specific {observability} alert indices or disable all alerts and remove the Alerts page altogether.
+You can do this in {kib} settings.
+
+If you are using our https://www.elastic.co/cloud/elasticsearch-service[hosted {ess}] on {ecloud},
+you'll edit the {kib} user settings:
+
+. Select your deployment on the home page, and from your deployment menu go to the *Edit* page.
+. In the *{kib}* section, click *Edit user settings*, and add the desired settings (detailed below).
+. Click *Back*, and then click *Save*. The changes are automatically appended to the `kibana.yml` configuration file for your instance.
+
+If you have a self-managed {stack}, you'll edit the settings in your `kibana.yml` file.
+
+[discrete]
+[[create-alerts-disable-some]]
+=== Disable writing to specific alert indices
+
+To disable writing to specific {observability} alerts-as-data indices
+while continuing to write to others, use `xpack.ruleRegistry.write.disabledRegistrationContexts`.
+
+You can disable writing to alert indices for:
+
+* Logs (`observability.logs`)
+* Metrics (`observability.metrics`)
+* APM (`observability.apm`)
+* Uptime (`observability.uptime`)
+
+NOTE: Disabling writing to uptime alert indices will affect _all_ uptime rule types including monitor status and TLS rule types.
+
+For example, to disable writing to Logs alert indices, you would add the following to your {kib} settings:
+
+[source,yml]
+----
+xpack.ruleRegistry.write.disabledRegistrationContexts : ['observability.logs']
+----
+
+To disable writing to both Logs and Uptime alert indices, you would use: 
+
+[source,yml]
+----
+xpack.ruleRegistry.write.disabledRegistrationContexts : ['observability.logs', 'observability.uptime']
+----
+
+[discrete]
+[[create-alerts-remove-page]]
+=== Remove the Alerts page
+
+To disable writing to all alert indices and remove the Alerts page from {kib} altogether, use the following settings:
+
+[source,yml]
+----
+xpack.ruleRegistry.write.enabled: 'false'
+xpack.observability.unsafe.cases.enabled: 'false'
```

Comment:
@colleenmcginnis We can get your current PR merged, but I put a note to investigate further why we have `xpack.observability.unsafe.cases.enabled: false` as a requirement to disable Alerts page. I don't think it is needed. `xpack.observability.unsafe.cases.enabled` is used to hide or not the `Cases` page. @vinaychandrasekhar I would like your thoughts on that.

Reply:
I agree that alerts can be disabled without disabling cases (although doing that would be a corner case). cc @cnasikas as FYI and for any further feedback.

Reply:
Hi @mgiota and @vinaychandrasekhar! Indeed Cases are GA and not connected directly to the alerts. However, you can attach an alert to a case and view the alert's details flyout from within the Case. So if Cases are enabled and the alerts are disabled you cannot attach an alert to a case. If Cases are disabled and the alerts are enabled then you have to hide the "attach to a case" functionality from the alerts table. Another scenario that could occur (very unlikely tbh) is to enable both, attach an alert to a case, and then disable the alerts. The case will have an alert attached to a case but I am not sure if it can view the alert's details flyout because it is provided by the o11y.

Reply:
Thanks @cnasikas for the description. Based on that, I'd suggest still treating the two flags separately like @mgiota said at the start of this comment thread. Users can disable just alerts.